src/InContextSignup.js

```diff
@@ -54,13 +54,18 @@ export class InContextSignup {
         const activeInput = this.device.activeForm?.activeInput
         activeInput?.blur()
 
-        // Focus on the active input to open the tooltip
-        const focusActiveInput = () => {
+        // Select the active input to open the tooltip
+        const selectActiveInpout = () => {
```

Comment:
Not sure if it's ready for review, but I've noticed the typo in `selectActiveInpout`.